components/GetstartedCard/index.tsx

```diff
@@ -0,0 +1,105 @@
+import classNames from 'classnames'
+import Styles from './index.module.scss'
+import {
+  completeYourIntegrationData,
+  startIntegratingData,
+  testAndGoData,
+} from './data'
+import LzLink from 'components/UI/Link'
+import LinkIcon from 'public/icons/link-icon'
+import ArrowRightIcon from 'public/icons/arrow-right-icon'
+
+// card with Icon component
+export const LinkedCard = ({ desc, title, route }) => {
+  return (
+    <div key={route}>
+      <LzLink className='my-2' to={route}>
+        <div
+          key={route}
+          className='bg-white text-sm mb-2 p-6 md:pt-9 drop-shadow-xs rounded-8 lg:rounded-8 h-[210px] lg:w-92 z-10'
```

Comment:
Please check this file and you'll see how we have setup typography to help us easily render texts easily. https://github.com/LazerPay-Finance/Lazerpay-documentation-v2/blob/main/styles/typography.scss

We don't pass the size or font directly: see this file for how its been used https://github.com/LazerPay-Finance/Lazerpay-documentation-v2/blob/a84ba68dce671c883c2872c1d53d58815474a843/components/QuickLinks/index.tsx#L22

Please update other places you entered the font weight or size directly(eg `text-sm` or `font-bold` :))

Reply:
fixed